parent/pom.xml

```diff
@@ -1412,6 +1412,7 @@
             <configuration>
               <excludes>
                 <exclude>**/*RandomizedPropertyTest.java</exclude>
+                <exclude>**/*$*.java</exclude> <!-- Re-add the default exclude for inner classes, see https://github.com/camunda-cloud/zeebe/issues/8637 -->
```

Comment:
:upside_down_face: We typically add the comments above the line in POMs, mostly for readability